packages/config/index.js

```diff
@@ -1,5 +1,5 @@
 if (process.env.CYPRESS_INTERNAL_ENV !== 'production') {
-  require('@packages/ts/register')
+  require('@packages/ts/registerDir')(__dirname)
```

Comment:
doesn't this same change need to be made everywhere we use `require('@packages/ts/register')` for full benefits?

Reply:
Created ticket UNIFY-1548 to apply this to all packages.